packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx

```diff
@@ -56,6 +69,10 @@ export const MultiValuesInput: React.FunctionComponent<MultiValuesInputProps> =
                                 parentProps.removeBox(cData.id);
                             }
                         }}
+                        placeholder={index >= limit ? reachedLimitPlaceholder : placeholder}
```

Comment:
Should we add someone from UX to review the style? It seems that placeholder has a large space on the top and left :thinking: 
![image](https://user-images.githubusercontent.com/52677246/89301253-756f5f80-d637-11ea-8568-2b0a1b2c3ff6.png)

Reply:
Yes! I'm not even sure that this placeholder would be relevant, so an UX opinion would help answer these questions. Summoning @oliviertaillon

Reply:
Can you make the styling the same as the one when there is an error on validation?

Reply:
I tend to agree with Sylvie here.  Do we really want a warning as soon as the user has begun to fill the last available field?  All in All I would put this behavior optionnal, the warning appearing only if it is passed in the props.

Reply:
Before I commit my changes, I would like to confirm that the new style of the placeholder is what you expected (which is looks like the invalid-message of the input). All things consider, if we all agree with this configuration, I suggest that this props is still an optional one (but with this new style). 
![exampleOfReachedPlaceholder](https://user-images.githubusercontent.com/58052881/90158027-75b8da80-dd5c-11ea-8163-e6f027d5f6bd.PNG)

Reply:
You can remove the grey background but after this I confirm the styling ✅